src/main/groovy/org/transmartproject/batch/highdim/assays/AssayStepsConfig.groovy

```diff
@@ -53,7 +53,8 @@ class AssayStepsConfig implements StepBuildingConfigurationTrait {
                 delimiter: DELIMITER_TAB,
                 names: ['studyId', 'siteId', 'subjectId', 'sampleCd',
                         'platform', 'sampleType', 'tissueType', 'timePoint',
-                        'categoryCd', 'source_cd'],
+                        'categoryCd'],
+                strict: false
```

Comment:
Setting `strict` to false seems to olax:

> Public setter for the strict flag. If true (the default) then number of 
> in line must match the number of tokens defined 
> {@link Range}, columns, etc.) in {@link LineTokenizer}. 
> false then lines with less tokens will be tolerated and padded with 
> columns, and lines with more tokens will 
> simply be truncated.

So ot will also tolerate missing columns. And it will of course not only ignore `source_cd` but any number of extra columns, with any name. I think it's better to implement a `TokenizerColumnsReplacingHeaderHandler`, like it's done for clinical data.